gslib/wildcard_iterator.py

```diff
@@ -238,8 +238,8 @@ def __iter__(self,
             if obj_or_prefix.datatype == CloudApi.CsObjectOrPrefixType.OBJECT:
               gcs_object = obj_or_prefix.data
               if prog.match(gcs_object.name):
-                if not suffix_wildcard or (StripOneSlash(gcs_object.name)
-                                           == suffix_wildcard):
+                if not suffix_wildcard or (StripOneSlash(
+                    gcs_object.name) == suffix_wildcard):
```

Comment:
Assuming this isn't doing anything you can just remove this.

Reply:
Yep, these changes were made because I ran `python3 -m yapf -irp .`
But Travis CI failed anyway. Turns out, yapf behaves differently for python3.6 vs python3.7. Running in python3.6 removed this change so we should be good now.